src/encoded/types/file.py

```diff
@@ -163,9 +164,23 @@ def post_upload(context, request):
     return CorePostUpload(context, request)
 
 
+def validate_user_has_protected_access(request):
+    """ Validates that the user who executed the request context either is
+        an admin or has the dbgap group
```

Comment:
Should this also allow people from the file's SubmissionCenter(s) to download? Not sure if that's desirable, though it seems intuitive; probably at least the person who uploaded the file should have perms to download it?

Reply:
We do not allow that in 4DN - once a file is restricted status no one but admin can download it - however, use case may be different for SMaHT - we only set this status upon data release.

Reply:
is the dbgap group too general?  Is there additional control based on which study or does/will not that apply?

Reply:
Not for download but we do have a read-only-admin group but again not specific - anyone with that group can view - which is why I raised the question.
In 4DN if something is 'in review by lab' or 'uploaded' status then only a member of the Lab linked to a file/item can view/download it but that case is a bit different.

Reply:
The assumption I've made here is that users who are submitting protected data would have the `dbgap` group, and that there is only one set of files for this purpose, so a single group suffices. This does definitely differ from 4DN, those with access to the protected data will be able to download it, while it is still viewable but not downloadable by those who do not have access.

Reply:
may be relevant and this is something we don't use a lot and is kind of a pain but we do have 'viewing_groups'  and Andrea needed a mechanism to share a non-released item with a select group of people who otherwise did not share a lab, award or other attribute so there is a mixin property called 'viewable_by' that is a string that is matched with a user.viewing_group and allows view of that item.  Could potentially be adapted for download but is clunky and there is likely a better way.  

Will I'm not sure I understand - is the eventual plan to have the restricted data elsewhere for download through dbgap so this group only applies to submitters?  Or if not will all the SMaHT data fall under the same dbgap study so as long as a user is approved for dbgap they can download any restricted file on the portal?

Reply:
To my understanding, there will be 1 dbGaP study for SMaHT and it is binary choice whether they have it or not hence the single group. We will be storing this data ourselves and giving folks access to it through our portal, though I do think eventually moving data to SRA or ANViL was discussed previously as well.